### PR TITLE
Clean up xpi.sh

### PR DESCRIPTION
**Pull request by biancadanforth**
* Fixed typos in mustache alias.
  - Note: The mustache alias is not needed inside the shell script, only on the command line, since the $PATH environment variable for the shell scripts includes ./node_modules, but I left it here for command line debugging.
* Added addon.xpi symlink note to README.md
* Removed full variable expansion notation outside of strings for improved readability. -- happy to change this back if you prefer.
  - Quotes left around BASE_DIR since it could have spaces in it
  - Good resource: https://en.wikibooks.org/wiki/Bash_Shell_Scripting#Variables
* Fixed a couple other typos, like in the SUCCESS message.
* Learned how to read shell scripts :)

### CODE REVIEW

**Comment by biancadanforth** [comment]
@gregglind , please let me know what you think of these changes!

Also, I have a follow-up question for you:
1. As a sanity check, I compared the xpi content built before and after the variable expansion changes, and they differ-- though they have the same file size, to the byte. I then found that the xpi content built between `npm run build` for the exact same code also differs. Why is this?

**Comment by gregglind** [comment]
I already pulled out make_xpi.  Looking over your suggestions.

Re: npm run build, will investigate.  You can use `tar -xO -f addon.xpi` if you want to see what is in there.  I don't know which phases are fully deterministic, and which aren't.

File: README.md [hunk 1]
```diff
@@ -5,6 +5,8 @@
 `npm install`
 `npm run build`
 
+Note: `addon.xpi` is a symbolic link to the extension's true XPI, which is named based on the study's unique addon ID specified in `package.json`.
```
Comment (gregglind) [inline README.md]: solved this with both a note AND changing the name to 'linked-addon.xpi'

File: bin/xpi.sh [hunk 2]
```diff
@@ -8,44 +8,40 @@ set -eu
...
 function cleanup {
-  rm -rf "$TMP_DIR"
+  rm -rf $TMP_DIR
 }
 trap cleanup EXIT
 
 # fill templates, could be fancier
 echo $PWD
-alias moustache='/node_modules/bin/mustache'
+alias mustache='/node_modules/bin/mustache'
```
Comment (gregglind) [inline bin/xpi.sh]: the alias here is necessary if you aren't calling it from `npm`.  Only under npm do they add node_modules/.bin to the path.

File: bin/xpi.sh [hunk 3]
```diff
@@ -8,44 +8,40 @@ set -eu
...
 mustache package.json template/install.rdf.mustache > addon/install.rdf
 mustache package.json template/chrome.manifest.mustache > addon/chrome.manifest
 
 cp -rp addon/* $DEST
 
 pushd $DEST
-zip -r $DEST/${XPI_NAME} *
-mkdir -p $BASE_DIR/dist
-mv "${XPI_NAME}" $BASE_DIR/dist
+zip -r $DEST/$XPI_NAME *
```
Comment (gregglind) [inline bin/xpi.sh]: I actually went through and quoted them all, for safety.  Anything is possible.

**Pull request closed without merging**